app/controllers/api/v1/interactive_pages_controller.rb

```diff
@@ -82,8 +90,12 @@ def update_page
       @interactive_page.update_attributes({
         name: page_params['name'],
         is_completion: page_params['isCompletion'],
-        is_hidden: page_params['isHidden']
+        is_hidden: page_params['isHidden'],
```

Comment:
Seems like the comma at the end of this line isn't necessary. Should it be removed?